doc/api/deprecations.md

```diff
@@ -999,6 +999,15 @@ Type: Documentation-only
 The option `produceCachedData` has been deprecated. Use
 [`script.createCachedData()`][] instead.
 
+<a id="DEP00XX"></a>
+### DEP00XX: process.binding()
+
+Type: Documentation-only
+
+The `process.binding()` API is intended for use strictly by Node.js internal
+code to provide a bridge between Node.js' JavaScript and native code layer.
+Use of `process.binding()` by user-land code is unsupported.
```

Comment:
AFAIK, we use `userland` everywhere and `user-land` nowhere, so let's stick with `userland`.